### PR TITLE
PSR-206 scheme hold land property

### PR DESCRIPTION
**Pull request by narendrapaduchuri**

### CODE REVIEW

File: app/pages/nonsipp/landorproperty/WhyDoesSchemeHoldLandPropertyPage.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+package pages.nonsipp.landorproperty
+
+import models.SchemeHoldLandProperty
+import models.SchemeId.Srn
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case class WhyDoesSchemeHoldLandPropertyPage(srn: Srn) extends QuestionPage[SchemeHoldLandProperty] {
+
+  override def path: JsPath = JsPath \ toString
```
Comment (kberec) [inline app/pages/nonsipp/landorproperty/WhyDoesSchemeHoldLandPropertyPage.scala:26]: Data path would be similar to the one commented on the ticket.
We have a similar example in the loans (with one element less), where saved off data looks like (key 0/1/2/3 are indexes of the elements - from the URL):
```
  "loans": {
    "loanTransactions": {
      "recipientIdentityType": {
        "whoReceivedLoans": {
          "0": "individual",
          "1": "ukCompany",
          "2": "ukPartnership",
          "3": "other"
        },
```

File: app/pages/nonsipp/landorproperty/WhyDoesSchemeHoldLandPropertyPage.scala [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+package pages.nonsipp.landorproperty
+
+import config.Refined.Max9999999
+import models.SchemeHoldLandProperty
+import models.SchemeId.Srn
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+import utils.RefinedUtils.RefinedIntOps
+
+case class WhyDoesSchemeHoldLandPropertyPage(srn: Srn, index: Max9999999) extends QuestionPage[SchemeHoldLandProperty] {
```
Comment (westwater) [inline app/pages/nonsipp/landorproperty/WhyDoesSchemeHoldLandPropertyPage.scala:26]: index should be 5000 but i'll add it in my ticket 👍

**Review by westwater — DISMISSED** [review]

File: app/pages/nonsipp/landorproperty/WhyDoesSchemeHoldLandPropertyPage.scala (hunk 2 above)
Comment (narendrapaduchuri) [reply app/pages/nonsipp/landorproperty/WhyDoesSchemeHoldLandPropertyPage.scala:26]: thank you

**Review by kberec — DISMISSED** [review]

**Comment by westwater** [comment]
@narendrapaduchuri can this be closed now?

**Pull request closed without merging**